src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Log/FunctionId.cs

```diff
@@ -493,5 +493,7 @@ internal enum FunctionId
         DependentTypeFinder_FindAndCacheDerivedClassesAsync = 430,
         DependentTypeFinder_FindAndCacheDerivedInterfacesAsync = 431,
         DependentTypeFinder_FindAndCacheImplementingTypesAsync = 432,
+
+        LSPCompletion_MissingLSPCompletionTriggerKind = 433,
```

Comment:
this will need to be GDPR approved, ping me when this is merged if you don't know how

Reply:
Will do! (I think you've actually shared the link to the GDPR telemetry site with me before, but I've lost it. 😞 )

Also, thanks for the review - switching from using arrays to sets really cleaned up the code in some places.